cli/stubs/parsy.pyi

```diff
@@ -0,0 +1,116 @@
+from typing import TypeVar
+from typing import Generic
+from typing import Callable
+from typing import Iterable
+from typing import Tuple
+from typing import List
+from typing import Optional
+from typing import Protocol
+from typing import overload
+from enum import Enum
+
+from re import Pattern
+
+T = TypeVar("T", covariant=True)
+A = TypeVar("A")
+B = TypeVar("B")
+
```

Comment:
I have no idea what those I/O/... mean.
Would be nice to add some comments.

Reply:
Or did you copy this file from the web? In that case could be nice to reference where it comes from?

Reply:
No I wrote this myself, by searching through mypy docs whenever I hit seemingly untype-able functions. I can write some explanations though